spec/features/data_access/direct_download_sit_spec.rb

```diff
@@ -1,12 +1,20 @@
 require 'rails_helper'
 
 describe 'Direct download script SIT' do
-  context 'when viewing the direct download script in SIT', pending_updates: true do
+  context 'when viewing the direct download script in SIT' do
     before :all do
       load_page 'projects/new', env: :sit, project: ['C1000000082-EDF_OPS'], authenticate: 'edsc'
       wait_for_xhr
 
-      page.find('.button-download-data').click
+      collection_card = find('.project-list-item', match: :first)
+      collection_card.find('.project-list-item-action-edit-options').click
+      wait_for_xhr
+
+      click_button('Edit Delivery Method')
+
+      choose('Direct Download')
+
+      page.find_button('Download Data', disabled: false).click
```

Comment:
What does `disabled: false` do here? Wait until it's enabled?

Reply:
Checks the disabled attribute on the element

Reply:
Sorry, my question was more.. why is this required. I assume to wait until the button is enabled since there are calls in the background that take time to enable it?

Reply:
I probably just copied the code chunk from a test that was testing if the button was re-enabled after a bad echoform was used. I'll try it without

Reply:
ah, I think that's the reason then haha